### PR TITLE
Added link to Arxiv paper

### LINKED ISSUE #112: Add link to Arxiv preprint in readme
Please consider mentioning and adding a link to the corresponding Arxiv preprint https://arxiv.org/abs/1906.05264 in the readme of this repo. Thanks.

### PR DESCRIPTION
**Pull request by jaheba**
Fixes #112

### CODE REVIEW

**Comment by alexw91** [comment]
# [Codecov](https://codecov.io/gh/awslabs/gluon-ts/pull/117?src=pr&el=h1) Report
> Merging [#117](https://codecov.io/gh/awslabs/gluon-ts/pull/117?src=pr&el=desc) into [master](https://codecov.io/gh/awslabs/gluon-ts/commit/dd6784bfae9becce1d9c0f7da891b01544a86fd0?src=pr&el=desc) will **not change** coverage.
> The diff coverage is `n/a`.


[Impacted file tree graph](https://codecov.io/gh/awslabs/gluon-ts/pull/117?src=pr&el=tree)

```diff
@@           Coverage Diff           @@
##           master     #117   +/-   ##
=======================================
  Coverage   79.05%   79.05%           
=======================================
  Files         110      110           
  Lines        6301     6301           
=======================================
  Hits         4981     4981           
  Misses       1320     1320
```

**Comment by szha** [comment]
Job PR-117/1 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-117/1/index.html

File: README.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ state of the art models ready to be trained, and building blocks to define
 your own models and quickly experiment with different solutions.
 
 * [Documentation](https://gluon-ts.mxnet.io/)
+* [Paper](GluonTS: Probabilistic Time Series Models in Python)
```
Comment (jgasthaus) [inline README.md]: I think you want to add the actual link here: https://arxiv.org/abs/1906.05264

Comment (jaheba) [reply README.md]: Of course. Thanks!

**Comment by szha** [comment]
Job PR-117/2 is complete. 
Docs are uploaded to http://gluon-ts-staging.s3-accelerate.dualstack.amazonaws.com/PR-117/2/index.html

**Pull request merged**